kaldb/src/test/java/com/slack/kaldb/server/KalDbIntegrationTest.java

```diff
@@ -0,0 +1,84 @@
+package com.slack.kaldb.server;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.slack.kaldb.config.KaldbConfig;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.HashMap;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KalDbIntegrationTest {
+  private static final Logger LOG = LoggerFactory.getLogger(KalDbIntegrationTest.class);
+
+  private Kaldb kaldb;
+  private final ObjectMapper om = new ObjectMapper();
+
+  @Before
+  public void start() throws IOException {
+    KaldbConfig.reset();
+    kaldb = new Kaldb(Path.of("../config/config.yaml"));
+    LOG.info("Starting kalDb with the resolved configs: {}", KaldbConfig.get().toString());
```

Comment:
Should we move this statement into KaldbConfig when it gets initialized? It would be very useful to print the config when an application starts.

Reply:
The difficulty in putting this into the main application is ensuring that we don't print out any secrets into the logs. As we currently don't have a way to mark specific values as secret, we should keep this behavior to the tests.